src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -102,6 +110,49 @@ internal sealed class SourcePropertyAccessorSymbol : SourceMemberMethodSymbol
                 diagnostics);
         }
 
+#nullable enable
+        public static SourcePropertyAccessorSymbol CreateAccessorSymbol(
+            bool isGetMethod,
+            bool usesInit,
+            NamedTypeSymbol containingType,
+            SynthesizedRecordPropertySymbol property,
+            DeclarationModifiers propertyModifiers,
+            string propertyName,
+            Location location,
+            CSharpSyntaxNode syntax,
+            DiagnosticBag diagnostics)
+        {
+            string name;
+            ImmutableArray<MethodSymbol> explicitInterfaceImplementations;
+            GetNameAndExplicitInterfaceImplementations(
+                explicitlyImplementedPropertyOpt: null,
+                propertyName,
+                property.IsCompilationOutputWinMdObj(),
+                aliasQualifierOpt: null,
+                isGetMethod,
+                out name,
+                out explicitInterfaceImplementations);
+
+            var methodKind = isGetMethod ? MethodKind.PropertyGet : MethodKind.PropertySet;
+            return new SourcePropertyAccessorSymbol(
+                containingType,
+                name,
+                property,
+                propertyModifiers,
+                explicitInterfaceImplementations,
+                location,
+                syntax,
+                hasBody: false,
```

Comment:
>false [](start = 25, length = 5)

Why false? I think this is supposed to reflect if there will be a body for the accessor rather than just if there is one in the syntax.

Reply:
Ok, I think ```isAutoPropertyAccessor: true``` should give us the right behavior. But we should test with data properties declared in interfaces.

---
In reply to: [445683605](https://github.com/dotnet/roslyn/pull/45411#discussion_r445683605) [](ancestors = 445683605)

Reply:
Right, this is asking whether there's a body, not whether there's an implementation. Since these are auto-properties they have no body, but a compiler-provided implementation.

I agree that the data property PR should definitely have a test for a data property in an interface.